habitat_sim/utils.py

```diff
@@ -5,6 +5,72 @@
 # LICENSE file in the root directory of this source tree.
 
 import numpy as np
+import quaternion
+
+
```

Comment:
General comment about functions in utils: let's add at least some minimal docstrings now that the functions are fresh. :-)

Reply:
On it